src/Raven.Studio/wwwroot/App/views/common/notificationCenter/detailViewer/licenseLimitDetails.html

```diff
@@ -20,9 +20,9 @@ <h3 class="modal-title" data-bind="text: licenseLimitNotification.title"></h3>
 
         <div class="modal-footer">
             <button type="button" class="btn btn-default" data-bind="click: $root.close">Close</button>
-            <a target="_blank" href="https://ravendb.net/buy" class="btn btn-primary">
+            <a target="_blank" data-bind="click: $root.register, attr: { 'href': url }" class="btn btn-primary">
```

Comment:
do we need both click and href?

Reply:
yes. need to open the registration window after the redirect

Reply:
Isn't href ignored when you have client handler?

Reply:
it's not if the handler returns true